shopify/README.md

```diff
@@ -0,0 +1,65 @@
+# Nimbella Commander Shopify Command Set
+A Nimbella Commander command set for viewing orders, adding notes to orders, getting numbers of products sold,  and getting inventory numbers. 
+
+## Commands
+- `orders` - Get a detailed list of orders .
+- `add_note <order_id> <note>` - Add a note to an existing order.
+- `inventory [<search_words>]` - Get inventory numbers for products.
+- `products_sold [<search_terms>]` - Get sale numbers for products in shop.
+- `shopify` - View command set documentation
+
+## Install
+```
+/nc csm_install shopify
+```
+
+## Requirements
+
+
+In order to use this command set, you must set up an app on Shopify with the following scopes
+```
+Inventory (Read access)
+Orders, transactions and fulfillments (Read and write)
+Products, variants and collections (Read)
+```
+Using /nc secret_create create the following keys
+```
+Key: shopifyKey / Value: API Key
+Key: shopifyPassword / Value: Password
+Key: shopifyHostname / Value: Hostname
+```
+
+## Usage
+
+View list of past orders
+```
+/dapp shopify_orders
```

Comment:
I guess we need to change `/dapp` to `/nc` now. cc @niks3089

Reply:
Yes, please replace `/dapp` with `/nc`

Reply:
Can do 👍